components/suggestion/at_mention_provider/at_mention_provider.test.jsx

```diff
@@ -451,18 +451,18 @@ describe('components/suggestion/at_mention_provider/AtMentionProvider', () => {
             component: AtMentionSuggestion,
         });
 
-        await Promise.resolve().then(() => {
-            expect(resultCallback).toHaveBeenNthCalledWith(3, {
-                matchedPretext,
-                terms: [
-                    '@user.six-split',
-                ],
-                items: [
-                    {type: Constants.MENTION_NONMEMBERS, ...userid6},
-                ],
-                component: AtMentionSuggestion,
-            });
-        });
+        // await Promise.resolve().then(() => {
```

Comment:
Are you intending to comment this out?

Reply:
Nope, earlier tests were failing so i had to comment it. Thanks for it. Un commented it